app/views/applications/applications/_api_credentials.html.erb

```diff
@@ -1,10 +1,9 @@
-<% if cinstance.service.backend_version == "2"  %>
```

Comment:
I guess this is removed because it is redundant?
https://github.com/3scale/porta/blob/c44891a88fb3f9f29eeaada3cb39a165c4a3f348/app/views/applications/access_details/show.html.erb#L4-L5

Reply:
Correct